process_creationg_termination.c

```diff
@@ -16,19 +15,17 @@ int main()
     childPid = fork();
     if(childPid  >= 0)
     {
-        if(childPid == 0)
-        {
-            printf("Child process\n");
-            sleep(5);
+        if(childPid == 0) {
+            printf("Child greeting\n");
+            sleep(2);
             printf("Child exiting\n");
         }
-        else
-        {
-            waitpid(-1,NULL,0);
+        else{
+           sleep(2);
```

Comment:
Child and parent process execute independently after being forked imo, please correct me if i'm wrong
http://www.csl.mtu.edu/cs4411.ck/www/NOTES/process/fork/create.html